pkg/util/filehelper/file.go

```diff
@@ -152,3 +152,20 @@ func CopyDir(src string, dst string) (err error) {
 
 	return
 }
+
+// IsDirEmpty checks whether directory is empty or not
```

Comment:
What is the behaviour if the directory does not exist or is a file which is not a directory?